.github/workflow/publish-ecr.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish-ECR
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  deploy:
+    name: Publish to Amazon ECR
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
```suggestion
          aws-access-key-id: ${{ secrets.ECR_ACCESS_KEY_ID }}
```

Reply:
We set up org-wide secrets called ECR_...

Reply:
Great! thanks